remediation/machine-deletion/machine-deletion.md

```diff
@@ -10,6 +10,4 @@ parent: Remediation
 Generally Available
 {: .label .label-green }
 
-This operator conforms to the External Remediation of [NodeHealthCheck](https://github.com/medik8s/node-healthcheck-operator#readme) and is designed to work with it in order to reprovision unhealthy nodes using the [Machine API](https://github.com/openshift/machine-api-operator#readme). 
-It looks for the associated Machine of an unhealthy Node, and deletes it.  
-Once the Machine CR has been deleted, the owning controller creates a replacement. 
+[Machine Deletion Remediation](https://github.com/medik8s/machine-deletion#readme) is an External Remediation of [Node Health Check](https://github.com/medik8s/node-healthcheck-operator#readme), designed to reprovision unhealthy nodes using the [Machine API](https://github.com/openshift/machine-api-operator#readme).
```

Comment:
IIUC `External Remediation` is a term that is relevant in the context of MHC and not NHC

Reply:
Maybe something like:
 `Machine Deletion Remediation` is a remediator that can be used by `Node Health Check` ... :thinking:

Reply:
ok, simple and clean